main.tf

```diff
@@ -40,6 +40,9 @@ resource "aws_db_instance" "main_rds_instance" {
 
     backup_retention_period = "${var.backup_retention_period}"
     backup_window = "${var.backup_window}"
+    tags {
+        "Name" = "Test"    
```

Comment:
Quotes around "Name" are not needed.
Also: How this is useful? Maybe use ${var.database_name} or something?